tests/ui/const-generics/issues/issue-100313.rs

```diff
@@ -7,14 +7,13 @@ impl<const B: &'static bool> T<B> {
     const fn set_false(&self) {
         unsafe {
             *(B as *const bool as *mut bool) = false;
```

Comment:
```suggestion
            *(B as *const bool as *mut bool) = false; //~ failed here
```
we'll likely want this in other tests, too, to ensure that the failure is actually about the thing we want to test